src/frontend/Felles/Personopplysninger/Adressehistorikk.tsx

```diff
@@ -84,7 +84,7 @@ const Kolonnetittel: React.FC<{ text: string; width: number }> = ({ text, width
 );
 
 const TittelbeskrivelseBostedsadresser: React.ReactElement = (
-    <HelpText title="Gjeldende og kolonner" placement={'right'}>
+    <>
```

Comment:
Trenger vi fragment het? :)

Reply:
ah, ser det er fikset i en senere commit

Reply:
Hehe ja commit for commit funker kanskje ikke like bra for alle steder, men for de to første hvertfall!